server/app/views/applicant/NorthStarApplicantUpsellView.java

```diff
@@ -48,6 +52,14 @@ public String render(UpsellParams params) {
     context.setVariable("applicationId", params.applicationId());
     context.setVariable("bannerMessage", params.bannerMessage());
 
+    String alertTitle =
+        params
+            .messages()
+            .at(MessageKey.ALERT_SUBMITTED.getKeyName(), params.programTitle().orElse(""));
+    AlertSettings successAlertSettings =
+        new AlertSettings(true, Optional.of(alertTitle), "", AlertType.SUCCESS, ImmutableList.of());
```

Comment:
**suggestion (non-blocking):** Could we add a comment with what the boolean is? Makes it easier when reading :)